edb/edgeql/compiler/options.py

```diff
@@ -66,6 +66,14 @@ class GlobalCompilerOptions:
     #: definitions.
     func_params: Optional[s_func.ParameterLikeList] = None
 
+    #: The name that can be used in a "DML is disallowed in ..."
+    #: error. When this is not None, any DML should cause an error.
+    in_ddl: Optional[str] = None
```

Comment:
I'd rename to something like

```suggestion
    in_ddl_context_name: Optional[str] = None
```

Reply:
Renamed